### PR TITLE
Various documentation fixes/improvements

### PR DESCRIPTION
**Pull request by joshurtree**
Fixed links to point to stt-datacore repository, added section about setting up the server to use the asset server and add back lost ways to help page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,16 +19,16 @@ A Linux VM with 2 CPUs and 4Gb of dedicated RAM should suffice for the current l
...
 
 #### assets
-This VM runs a cronjob every 10 minutes that scans, downloads and unpacks new assets (crew images) to the local file system. There's also an nginx HTTP server that publishes the assets. Code is hosted [here](https://github.com/TemporalAgent7/asset-server).
+This VM runs a cronjob every 10 minutes that scans, downloads and unpacks new assets (crew images) to the local file system. There's also an nginx HTTP server that publishes the assets. Code is hosted [here](https://github.com/stt-datacore/asset-server).
 
 #### Image analysis
-Written in dotnet core and using OpenCV and Tesseract OCR, this is the most taxing (CPU and RAM) component of the system, used by the bot for recognizing behold (and voyage setup) screenshots. Code is hosted [here](https://github.com/TemporalAgent7/datacore-bot).
+Written in dotnet core and using OpenCV and Tesseract OCR, this is the most taxing (CPU and RAM) component of the system, used by the bot for recognizing behold (and voyage setup) screenshots. Code is hosted [here](https://github.com/stt-datacore/bot).
```
Comment (AlexCPU) [inline README.md:25]: This should point to the cpp-image-analysis repo. And it's now written in c++ with the old dotnetcore deprecated.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -14,10 +14,16 @@ If you want to contribute code fixes, see below.
...
-$ git clone https://github.com/TemporalAgent7/datacore
+$ git clone https://github.com/stt-datacore/website.git
 $ npm install 
 ```
 
+In order to get images on your server you have to run the command. 
+```sh
+$ cp .env.default .env
+```
+Then change the line "GATSBY_ASSETS_URL=" to "GATSBY_ASSETS_URL=http://assets.datacore.app/".
```
Comment (AlexCPU) [inline CONTRIBUTING.md:25]: This should probably suggest changing it to point to wherever the person is hosting asset-server, or if they're pointing to us, it should be an https link

File: README.md [hunk 3]
```diff
@@ -19,16 +19,16 @@ A Linux VM with 2 CPUs and 4Gb of dedicated RAM should suffice for the current l
 Written in TypeScript with React.js and built with Gatsby, combines source code with big book notes (as markdown) and other data (as json) to create a static website which needs to be uploaded to the server's file system (CI loop recommended for build validation and deployments).
```
Comment (AlexCPU) [inline README.md:19]: A comment that this repo is the website content maybe?

File: README.md [hunk 4]
```diff
@@ -19,16 +19,16 @@ A Linux VM with 2 CPUs and 4Gb of dedicated RAM should suffice for the current l
...
 #### site-server
-Serves the dynamic aspects of datacore.app (profile uploads / views, fleet info, crew comments). Code is hosted [here](https://github.com/TemporalAgent7/site-server).
+Serves the dynamic aspects of datacore.app (profile uploads / views, fleet info, crew comments). Code is hosted [here](https://github.com/stt-datacore/site-server).
 
 #### discord bot
-The Discord bot implementation (written in TypeScript with discord.js). See source code [here](https://github.com/TemporalAgent7/datacorebot).
+The Discord bot implementation (written in TypeScript with discord.js). See source code [here](https://github.com/stt-datacore/bot).
 
 #### DB
 A simple DB (currently PostgreSQL but configurable) that links discord user ids with uploaded profiles (where associated) and includes the crew comments.
```
Comment (AlexCPU) [inline README.md:34]: We're lazy and using SQLite now, as the weight of postgresql wasn't warranted for the amount of data we hold (precious little)

**Review by AlexCPU — CHANGES_REQUESTED** [review]
In the second paragraph on contributing.md (it's not letting me submit a comment for some reason) well above any of your edits. This could do with completely rewriting to say that we don't accept content changes to crew data, everything represents how it is displayed in game, or is sourced from the big book.

Thanks for tackling this, I know how much people dislike documenting their work!

File: README.md (hunk 4 above)
Comment (joshurtree) [reply README.md:34]: That being the case then the assets.svg needs to be altered as well as it currently also states that we use postgresql.

File: CONTRIBUTING.md (hunk 2 above)
Comment (joshurtree) [reply CONTRIBUTING.md:25]: My thinking is that the vast majority of potential contributors won't have any interest in the asset server. Its a work horse where nothing sexy happens. Plus this document is about how to contribute to the website. Ideally there should be a parallel document on that repository explaining how to setup a local version. Then that can linked here in a second sentence. Until that time maybe adding a sentence like "If you are using a local version of the asset server then use "GASTBY_ASSETS_URL=local_dir" could work.

**Pull request merged**